### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JanitaRiyaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,5 +8,40 @@ All Routes requires these parameters: `session`, `semester`, `degree`, `section`
...
+ ### Complete documents about project  📘
+ - please read [how to Setup](Docs) before diving into project.
+ <br><br>
+
+### See any issue ? ☠
+Create an issue with the correct label.
+<br><br>
+
+### For folks who wants to contribute ❤
+- please read [how to contribute](CONTRIBUTING.md) before getting started.
```
Comment (IIvexII) [inline README.md:31]: Please replace CONTRIBUTING.md with the link https://github.com/IIvexII/LGU-TimetableAPI/blob/main/CONTRIBUTING.md

File: README.md [hunk 2]
```diff
@@ -8,5 +8,40 @@ All Routes requires these parameters: `session`, `semester`, `degree`, `section`
...
+
+<p align="left"> 
+
+ <img src="https://img.shields.io/badge/JavaScript-323330?style=for-the-badge&logo=javascript&logoColor=F7DF1E">
+ <img src="https://img.shields.io/badge/shell-1572B6?style=for-the-badge&logo=shell&logoColor=white">
+ 
+ <br>
+ 
+ ### Complete documents about project  📘
+ - please read [how to Setup](Docs) before diving into project.
```
Comment (IIvexII) [inline README.md:23]: Please replace the Docs with this link https://github.com/IIvexII/LGU-TimetableAPI/blob/main/docs/Setup_Guide.md

File: README.md (hunk 1 above)
Comment (JanitaRiyaz) [reply README.md:31]: ok

**Comment by IIvexII** [comment]
Great! You have done this job beyond my expectations. I have pointed out some issue please resolve them make a commit so that I can merge it.

**Review by IIvexII — COMMENTED** [review]
Great! Please resolve the first issue as well mention here

**Comment by JanitaRiyaz** [comment]
> Great! You have done this job beyond my expectations. I have pointed out some issue please resolve them make a commit so that I can merge it.

Done!

**Review by IIvexII — COMMENTED** [review]
This issue is not resolve please, have a look at it

**Review by IIvexII — COMMENTED** [review]
Issue Resolved!

**Pull request merged**